docs/developer-guide/etendo-classic/how-to-guides/How_To_Create_a_Module.md

```diff
@@ -0,0 +1,209 @@
+
+A module is a piece of additional functionality that can be deployed
+optionally and independently on top of Etendo Classic. Examples of modules are:
+additional reports, additional windows, connectors, content packs
+(translations, chart of accounts, list of tax codes, product categories, etc).
+
+!!!info
+    For a detailed description on the Etendo Classic modularity concept, see the
+    [Modularity Guide](/concepts/Modularity_Concepts.md).
+
+This section explains only the standard module type, because that is most
+relevant for the other sections.
+
+All new development should be done as a part of a module. Only modules which
+are marked as _In Development_ can be used in the development process. A
+module can consist of the following software artifacts:
+
+  * Application dictionary components: all new application dictionary artifacts must be associated with a module. Application dictionary windows have a field that enables you to link an application dictionary component to a module. 
+  * Software resources: these are artifacts not defined in Etendo Classic itself but, for example, Java classes, libraries, web resources (images, etc.), configuration files, etc. To maintain the relationship with the module, software resources need to be located in the module's directory inside the Etendo Classic development project. Each module has its own unique directory. 
+  * Reference data: modules can be distributed with their own set of reference business data, for example, product categories, tax code, etc. The reference data is defined in datasets which can be related to a module. 
+
+The process of developing a module has three main steps:
+
+  1. Register your module in the Application Dictionary and in the central repository. 
+  2. Develop the artifacts included in your module. Depending on the functional specification and technical design of your module, it might include only one type of artifacts or a combination of them. In following sections, each type of artifact is described in detail. 
+  3. Publish the module in the central repository. 
+
+!!!Important
+    Every piece of Etendo Classic code belongs to a module, including
+    Etendo Classic core itself. You should do all your developments through
+    modules, including customizations. You can make changes directly in
+    other modules -including Etendo Classic core- but it is highly recommended not
+    to do that. It makes it much easier to maintain Etendo Classic if you restrict
+    code changes to modules.
+
+A module can be distributed and downloaded by other Etendo Classic users via
+the central repository. For more information about the central repository and
+distribution of modules, visit the [Modularity Guide](/concepts/Modularity_Concepts.md).
+
+The following sections discuss the main topic of this section: create and
```

Comment:
Link to https://docs.etendo.software/latest/developer-guide/etendo-classic/how-to-guides/how-to-publish-modules-to-github-repository/

Reply:
Added link